src/main/webapp/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
```

Comment:
Move this call into your page's JavaScript file the way I had Toba do. See his previous PR and ask him about specifics if you need. Basically, I had him load the script at the bottom of the HTML instead of in the head, then I had him call functions like these from inside that file instead of waiting for the load event.